node-scripts/src/pipeline3/scripts.js

```diff
@@ -11,25 +11,35 @@ const heredoc_3 = cst.HEREDOC_3;
 const utils = require("../utils/utils");
 const model = require("../utils/model");
 const createEnvVarFile = function(instanceDef, dockerComposePath) {
```

Comment:
I don't think this function should be Docker specific, right?
Please rename to something more generic.